src/Sentry.Unity.Android/SentryJava.cs

```diff
@@ -151,7 +151,7 @@ public ScopeCallback(Action<AndroidJavaObject> callback) : base("io.sentry.Scope
                 }
                 catch (Exception e)
                 {
-                    UnityEngine.Debug.LogError($"Error in SentryJava.ScopeCallback: {e}");
+                    UnityEngine.Debug.LogWarning($"Error in SentryJava.ScopeCallback: {e}");
```

Comment:
I thought this was about using `UnityEngine.Debug` instead of the SentryLogger and not about LogError itself?

Reply:
ah, right, diagnostic logger doesn't send errors, right?